packages/web/src/pages/sign-in-page/SignInPageMobile.tsx

```diff
@@ -26,65 +30,63 @@ const messages = {
 
 export const SignInPageMobile = () => {
   return (
-    <Flex
-      className={styles.root}
-      direction='column'
-      w='100%'
-      h='100%'
-      justifyContent='space-between'
-      pb='4xl'
-    >
-      <Flex
-        className={styles.content}
-        w='100%'
-        pv='2xl'
-        ph='l'
-        direction='column'
-        gap='2xl'
-        justifyContent='space-between'
-      >
-        <Flex direction='column' gap='2xl' alignItems='center'>
-          <IconAudiusLogoHorizontalColor />
-          <Text variant='heading' size='l' tag='h1' color='heading'>
-            {messages.title}
-          </Text>
-          <Box w='100%'>
-            <Form>
-              <Flex direction='column' gap='2xl' w='100%'>
-                <Flex direction='column' gap='l'>
-                  {/* TODO: replace old TextField */}
-                  <HarmonyTextField name='email' label={messages.emailLabel} />
-                  <HarmonyPasswordField
-                    name='password'
-                    label={messages.passwordLabel}
-                  />
-                </Flex>
-                <Flex direction='column' gap='l'>
-                  <Button iconRight={IconArrowRight} type='submit'>
-                    {messages.signIn}
-                  </Button>
-                  <Text color='heading' variant='body'>
-                    {/* TODO: link destination */}
-                    {messages.forgotPassword}
-                  </Text>
+    <SignOnContainerMobile>
```

Comment:
SignOn or SignIn? or is it used for sign-up and sign-in?

Reply:
Yep! Used it for both sign up/in, so I named it `SignOn`